src/Adapter/MSTest.CoreAdapter/Properties/TestExtensionTypesAttribute.cs

```diff
@@ -1,22 +1,17 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-
-using Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter;
-using Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities;
-
-[assembly: TypesToLoad(typeof(MSTestDiscoverer), typeof(MSTestExecutor))]
```

Comment:
Why did you remove the types from class ad added to AssemblyInfo.cs?

Reply:
It wasn't necessary here, but to match with https://github.com/microsoft/vstest/pull/2676.